fast_neural_style/generate.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function
+import argparse
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torchvision import datasets, transforms
+from torch.autograd import Variable
+
+import net as models
+import numpy as np
+import os
+import cv2
```

Comment:
Is really needed? I think all functionality you need is already implemented as `torchvision.transforms`.